sdcm/cluster.py

```diff
@@ -694,11 +694,9 @@ def _try_split_coredump(self, coredump):
             core_files.append(file_name)
             file_size = self._get_coredump_size(file_name)
             if file_size and file_size > COREDUMP_MAX_SIZE:
-                cnt = file_size / COREDUMP_MAX_SIZE
-                cnt += 1 if file_size % COREDUMP_MAX_SIZE > 0 else cnt
-                self.log.debug('Splitting coredump to {} files'.format(cnt))
-                res = self.remoter.run('sudo split -n {} {} {}.;ls {}.*'.format(
-                    cnt, file_name, file_name, file_name))
+                self.log.debug('Splitting coredump to {}G files'.format(COREDUMP_MAX_SIZE / 1024 / 1024 / 1024.0))
+                res = self.remoter.run('sudo split -b {} {} {}.;ls {}.*'.format(
```

Comment:
Please improve as follows:
1. move COREDUMP_MAX_SIZE to the method, there is no need to put in global scope since it is private for this method and not used in any place
2. Define COREDUMP_MAX_SIZE in GB, split understands sizes provided in GB

split help
```
-b, --bytes=SIZE        put SIZE bytes per output file
...
The SIZE argument is an integer and optional unit (example: 10K is 10*1024).
Units are K,M,G,T,P,E,Z,Y (powers of 1024) or KB,MB,... (powers of 1000).
```
3. Remove `_get_coredump_size`, it is not needed, split handles this, when the size of the file is less than asked

Reply:
Good suggestion!
Updated the PR.